api/Document.json

```
@@ -3947,10 +3947,10 @@
             "description": "<code>cut</code> command",
             "support": {
               "chrome": {
-                "version_added": false
+                "version_added": "43"
               },
               "chrome_android": {
-                "version_added": false
+                "version_added": "43"
```

```suggestion
                "version_added": "42"
```

Also, this change applies to webview_android below, too